src/components/BooksForm.jsx

```diff
@@ -34,10 +34,11 @@ const BooksForm = () => {
   const handleError = () => (state ? 'Please add a name from the book and an author' : '');
   return (
     <div className="divForm">
-      <form onSubmit={handleSubmit}>
-        <input type="text" placeholder="Book title" onChange={handleTitle} />
-        <input type="text" placeholder="author" onChange={handleAuthor} />
-        <button type="submit">Add book</button>
+      <h2 className="form-title">ADD NEW BOOK</h2>
+      <form onSubmit={handleSubmit} className="form">
+        <input className="input" type="text" placeholder="Book title" onChange={handleTitle} />
+        <input className="input" type="text" placeholder="author" onChange={handleAuthor} />
+        <button className="form-btn" type="submit">ADD BOOK</button>
       </form>
       <p>{handleError()}</p>
     </div>
```

Comment:
- [ ] We are required to match the design exactly. Kindly use the correct font styling for the app, to match the design.

  Link to the design: https://github.com/microverseinc/curriculum-react-redux/blob/main/bookstore/sneak_peek_v2_0.md

  ![image](https://user-images.githubusercontent.com/54848363/230666245-aa3ddcb5-7504-4537-87c5-105057ffcc7c.png)